data/core/units.cfg

```diff
@@ -247,7 +247,24 @@ The Dunefolk’s inquisitive and explorative nature does not preclude military s
+
+The segregation and alternating gender-dominance of Saurian society is an outgrowth of their clearly defined gender roles. It is the responsibility of the female to hunt and find food, skills which ultimately train them to be warriors; the skirmishers, flankers, and ambushers other races so fear. Males, meanwhile, are responsible for guarding the clutch — the eggs left by the females. While this leaves time for the males to develop and hone the arts of astrology, healing, and magic, it also exposes them to significant danger, as they are stationary targets for a Saurian clan's number one enemy — other Saurian clans.
+
+New Saurian clans are started when the proper astrological signs are read. Called a “hatching,” each female indicated by the conjunction selects a group of individuals with lower rank and leave their source clan. Frequently all females with a specific trait will be indicated, causing multiple clans to “hatch” at the same time. Selection is a simple process: no group leaving can be larger than any other, all the groups together cannot be larger than the group being left, and higher ranking allows a female to overrule another female's choice of who they take.
+
+Because of the rapid growth of saurian populations, these frequent splits in clans, and the fact that cannibalism is not taboo among Saurians violence is one of the defining features of the Saurian life. This limits the growth of the Saurian culture, limiting it to fits and starts- as much of their knowledge is passed by oral tradition and their possessions must be mobile.
+
+<header>text='Geography'</header>
+Saurians can live in many different areas, though swamps are by far their most common habitat.
+
+<header>text='Biology'</header>
+Saurians live spectacularly short lives by comparison to most of the other races of Wesnoth, reaching full adulthood within three years and often dying by the time they are 10 to 15 years old. By far, the most common cause of death is violence. Saurian females produce clutches of about 20 eggs roughly once a year, which creates constant population pressure and would stress most carnivores' food supply. Hunters and scavengers, Saurians have extremely strong jaws and have a very powerful digestive system with highly acidic fluids, making them capable of eating and digesting their entire prey including skin, teeth, horns and bones. Further they have no aversion to and readily eat carrion and commit cannibalism. Due to this, the males of the Saurian race have specialized in cold-magics to take advantage of their own cold-blooded nature."
```

Comment:
Sorry, missed this paragraph in previous review.

their entire prey -> their prey whole

"Further" -> "Further,".  The rest of the sentence needs a change too: "they have no aversion to" should be followed by a gerund, but "eat carrion" and "commit cannibalism" aren't gerunds. Maybe "Further, they have no aversion to eating carrion and even committing cannibalism, which are both regular sights."

The last sentence is ambiguous, specifically, whether "their own cold blooded nature" refers to their cannibalism or to their body temperature. I assume it means the latter?